sk0-front/src/app/auth/store/auth.reducer.ts

```diff
@@ -51,23 +51,7 @@ const authStore = createReducer(
   ),
   on(
     registerSuccess,
-    (state, action): AuthState => ({
-      ...state,
-      isLoading: false,
-      error: null
-    })
-  ),
-  on(
-    registerFailure,
-    (state, action): AuthState => ({
-      ...state,
-      isLoading: false,
-      error: action.message
-    })
-  ),
-  on(
-    registerSuccess,
-    (state, action): AuthState => ({
```

Comment:
Why are these things removed? I did not found the equivalent changes in other reducers. Is it a "breaking change"?

Reply:
Ouch sorry, thanks for the clarifications, the pr diff did not show me. I stand corrected. 🚀